examples/Cargo.toml

```diff
@@ -17,8 +17,10 @@ bcs = "0.1.3"
 futures = "0.3.24"
 futures-util = "0.3.26"
 hex = "0.4.3"
-linera-sdk = { version = "0.3.0", path = "../linera-sdk" }
-linera-views = { version = "0.3.0", path = "../linera-views", default-features = false }
+# linera-sdk = { version = "0.3.0", path = "../linera-sdk" }
+linera-sdk = { path = "../linera-sdk" }
+# linera-views = { version = "0.3.0", path = "../linera-views", default-features = false }
+linera-views = { path = "../linera-views", default-features = false }
```

Comment:
Not sure what's going on, but if I don't do this hack, the build inside the docker container fails with this error:
`Step 11/26 : RUN cargo chef cook --release --recipe-path recipe.json --target x86_64-unknown-linux-gnu
 ---> Running in fec1dd2a03b6
info: syncing channel updates for '1.69.0-x86_64-unknown-linux-gnu'
info: latest update on 2023-04-20, rust version 1.69.0 (84c898d65 2023-04-16)
info: downloading component 'cargo'
info: downloading component 'rust-std'
info: downloading component 'rustc'
info: installing component 'cargo'
info: installing component 'rust-std'
info: installing component 'rustc'
    Updating crates.io index
error: failed to select a version for the requirement `linera-sdk = "^0.3.0"`
candidate versions found which didn't match: 0.0.1
location searched: /opt/zefchain/build/linera-sdk
required by package `counter v0.1.0 (/opt/zefchain/build/examples/counter)`
    ... which satisfies path dependency `counter` (locked to 0.1.0) of package `linera-execution v0.0.1 (/opt/zefchain/build/linera-execution)`
    ... which satisfies path dependency `linera-execution` (locked to 0.0.1) of package `linera-chain v0.0.1 (/opt/zefchain/build/linera-chain)`
thread 'main' panicked at 'Exited with status code: 101', /usr/local/cargo/registry/src/index.crates.io-6f17d22bba15001f/cargo-chef-0.1.62/src/recipe.rs:204:27
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
The command '/bin/sh -c cargo chef cook --release --recipe-path recipe.json --target x86_64-unknown-linux-gnu' returned a non-zero code: 101
ERROR
ERROR: build step 0 "gcr.io/cloud-builders/docker" failed: step exited with non-zero status: 101

--------------------------------------------------------------------------------------

BUILD FAILURE: Build step failure: build step 0 "gcr.io/cloud-builders/docker" failed: step exited with non-zero status: 101
ERROR: (gcloud.builds.submit) build ad7da604-9af7-4f6a-8f8c-a2ea5e8d7030 completed with status "FAILURE"`

Reply:
The rust version is also not correct

Reply:
That's the thing, I have no idea why 😓
Ok I'll remove the commented lines